### ISSUE TITLE
Error in selector

### ISSUE THREAD

**Issue by pantersxx3** [darideveloper/facebook-groups-post-bot#3]
elenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"css selector","selector":"[aria-label="Photo/video"]"}

**Comment by pantersxx3** [comment]
solved, change "Photo" for "Foto".

**Comment by darideveloper** [comment]
No, "Foto" will only works if you facebook is in spanish
I hope to fix this  (work in any language) in the next update
